ggml/src/ggml-cpu/ggml-cpu.c

```diff
@@ -2197,6 +2197,7 @@ static int ggml_get_n_tasks(struct ggml_tensor * node, int n_threads) {
                 case GGML_UNARY_OP_HARDSWISH:
                 case GGML_UNARY_OP_HARDSIGMOID:
                 case GGML_UNARY_OP_EXP:
+                case GGML_UNARY_OP_GELU_ERF:
```

Comment:
It seems the implementation is multithreaded - move it down with `n_tasks = n_threads;`